src/sound/asio/sound.h

```diff
@@ -134,4 +134,14 @@ class CSound : public CSoundBase
     static long      asioMessages ( long selector, long value, void* message, double* opt );
 
     char* cDriverNames[MAX_NUMBER_SOUND_CARDS];
+
+    // Windows native MIDI
+#define MAX_MIDI_DEVS 4
```

Comment:
This isn't enough.  Make is 128?

Reply:
We could certainly make it 8 or 16 if necessary, but I can't imagine a use case for more than that, can you? It refers to actual MIDI _interface_ devices, such as a USB device or a card with MIDI ports, not to the number of MIDI devices connected to a MIDI interface (even that would be a modest number, surely, and at the MIDI level devices are distinguished by channel 1-16).

We need some kind of value for sizing the array of handles and limiting the loop index.

Reply:
Working on it... update tomorrow